doc/userdoc/contribute/templates_styleguides/styleguide.rst

```diff
@@ -0,0 +1,608 @@
+The NEST documentation style guide
```

Comment:
In my view, templates and style guides are very different types of documents. I would not combine them in one folder.

Reply:
yes, I tend to agree I will separate them